curriculum/challenges/english/15-javascript-algorithms-and-data-structures-22/learn-intermediate-fetch-promises-and-async-await-by-building-an-fcc-forum-leaderboard/643c347cf6143d087a07cc8a.md

```diff
@@ -0,0 +1,243 @@
+---
+id: 643c347cf6143d087a07cc8a
+title: Step 7
+challengeType: 0
+dashedName: step-7
+---
+
+# --description--
+
+Call your `fetchData()` function and check the console to see what the data looks like.
+
+# --hints--
+
+You should call the `fetchData()` function, after the `fetchData()` function is defined.
+
+```js
+assert.match(code, /}\s*fetchData\(\)/);
```

Comment:
wasn't able to pass with the current test. This seems to work though
```suggestion
assert.match(code, /};?\s*fetchData\(\)/);
```

Reply:
done